src/components/user-profile.html

```diff
@@ -136,9 +171,9 @@ <h3 class="div-center" style="margin: initial;padding-bottom: inherit;">User Pro
 
       expandCollapseCard() {
         this.profileExpanded ? this.profileExpanded = false :
-         this.profileExpanded = true;
+          this.profileExpanded = true;
         this.buttonText === 'Expand' ? this.buttonText = 'Collapse' :
```

Comment:
Would do the same here, as well:
```suggestion
        this.buttonText = !this.buttonText;
```